src/components/prompts/FeedbackForm.js

```diff
@@ -189,13 +189,13 @@ class FeedbackPrompt extends React.Component {
           <Form>
             <Form.Input
               { ...inputProps('currentSnap') }
-              label={ 'If amount for the CURRENT SNAP subsidy was wrong, what\'s the correct amount?' } />
+              label={ 'If amount for the CURRENT SNAP amount was wrong, what\'s the right amount?' } />
```

Comment:
OK, putting "amount" in the sentence 3 times sounds pretty weird. How about "If the CURRENT SNAP amount was wrong, what's the right amount?"

Reply:
lol, I completely missed that. Yeah, I can do that.

Reply:
(and for the other similar sentences below.) Thanks!